pkg/storage/segments/repairer.go

```diff
@@ -29,18 +30,37 @@ type Repairer struct {
 	ec       ecclient.Client
 	identity *identity.FullIdentity
 	timeout  time.Duration
+
+	// multiplierOptimalThreshold is the value that multiplied by the optimal
+	// threshold results in the maximum limit of number of nodes to upload
+	// repaired pieces
+	multiplierOptimalThreshold float64
 }
 
-// NewSegmentRepairer creates a new instance of SegmentRepairer
-func NewSegmentRepairer(log *zap.Logger, metainfo *metainfo.Service, orders *orders.Service, cache *overlay.Cache, ec ecclient.Client, identity *identity.FullIdentity, timeout time.Duration) *Repairer {
+// NewSegmentRepairer creates a new instance of SegmentRepairer.
+//
+// excessPercentageOptimalThreshold is the percentage to apply over the optimal
```

Comment:
nit -maybe this comment should go inside the function instead of above it. Just my own opinion. Do whatever you want here

Reply:
It explain what the parameters is.
If it's inside then it won't appear in the godoc.